junit5/src/main/java/sunstone/api/WithAzureArmTemplate.java

```diff
@@ -43,4 +44,32 @@
      * </ul>
      */
     Parameter[] parameters() default {};
+
+    /**
+     * Resource group that should be used for deployment. Expression is allowed, e.g. {@code abc-${var:default}-xyz} -
+     * var is resolved from system properties. Resources in a group share lifecycle and the group is deleted as it is for
+     * undeploy operation.
+     *
+     * If empty, {@code junit5.az.group} from {@code sunstone.properties} is used.
+     */
+    String group() default "";
```

Comment:
I as a tester when could I want to use it?

Reply:
This is different to AWS regions. Resources in a Azure resource group should share same lifecycle. Lets say I want to test EAP image with different databases. I will have suite level template deploying EAP once and then class level templates deploying DB. EAP and DB must be in different resource groups.